cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/stream/StreamAdminTest.java

```diff
@@ -65,6 +65,12 @@ public void testCreateExist() throws Exception {
 
     streamAdmin.create(otherStreamId);
     Assert.assertTrue(streamAdmin.exists(otherStreamId));
+
+    streamAdmin.drop(streamId);
+    Assert.assertFalse(streamAdmin.exists(streamId));
```

Comment:
Unfortunately, no. I couldn't reproduce this failure in a unit/integration test. I added this stuff here just for completeness.